changebot/github/tests/test_github_api.py

```diff
@@ -110,6 +110,25 @@ def test_urls(self):
         assert self.pr._url_pull_request == 'https://api.github.com/repos/fakerepo/doesnotexist/pulls/1234'
         assert self.pr._url_review_comment == 'https://api.github.com/repos/fakerepo/doesnotexist/pulls/1234/reviews'
         assert self.pr._url_commits == 'https://api.github.com/repos/fakerepo/doesnotexist/pulls/1234/commits'
+        assert self.pr._url_files == 'https://api.github.com/repos/fakerepo/doesnotexist/pulls/1234/files'
+
+    def test_has_modified(self):
+        mock = MagicMock(return_value=[{
```

Comment:
@bsipocz , this mocks the GitHub API response that `paged_github_json_request()` would have returned without actually pinging GitHub API. That function, in turn, is called by `has_modified()`, which is being tested here. GitHub API doc is at https://developer.github.com/v3/pulls/#list-pull-requests-files , whose return value is the basis of this test. Hope this clarifies the matter.